### PR TITLE
docs: removed default endpoint on command miactl context set CONTEXT …

### PR DESCRIPTION
**Pull request by alexBonadei**
## What this PR is for?

After have installed miactl and created a new context I realized that the context `endpoint` did not have a default. I had to set it manually with this value `https://console.cloud.mia-platform.eu` (that was the one documented as default).

### CODE REVIEW

**Comment by fredmaggiowski** [comment]
I'd rather actually implement the default, instead of changing the doc 🤔 

@JGiola what do you think about it? Is there a reason why the default is not applied?

**Comment by JGiola** [comment]
The doc is wrong, probably a copy paste from the holdover when miactl didn't have the context functionality.

It's correct to remove it from the doc pages because the current flag help text doesn't mention it.

I don't remember right now if the default value wasn't preserved in the context creation for a mistake or for a valid reason...

**Review by fredmaggiowski — APPROVED** [review]
LGTM

**Review by JGiola — CHANGES_REQUESTED** [review]
Before merging, can you remove all the other references to this default in the same file?

File: go.mod
Comment (JGiola) [inline go.mod:1]: Can you leave out the go.mod and go.sum from the commit, you are only changing doc files...

**Review by JGiola — CHANGES_REQUESTED** [review]

**Review by JGiola — APPROVED** [review]
LGTM

**Pull request merged**